engines/suppliers/Gemfile.lock

```diff
@@ -2,68 +2,93 @@ PATH
   remote: .
   specs:
     suppliers (0.0.1)
-      rails (~> 4.1.7)
+      rails (~> 4.2.5)
```

Comment:
Oh sorry! I didn't noticed that it was an engine `Gemfile.lock`... In engines `Gemfile` is useful only for testing purposes. @sauloperez please update it accordingly with the app Rails version by hand. We'll need to deal with this issue once we finally implement tests. a.k.a. `dummy` vs `whole app` test suites.